components/file_upload/file_upload.jsx

```diff
@@ -616,12 +616,18 @@ export default class FileUpload extends PureComponent {
                         }}
                     >
                         <a>
-                            {item.icon}
+                            {item.icon}&nbsp;
```

Comment:
Why use an nbsp here? I think we generally do this using a margin

Reply:
I'm sure we do, but I sort of added it to flag your attention : ). There needs to be space between the icon and the string, but I'm wondering where is the best place in the CSS to add it. This was more of a temporary fix as it was driving me nuts. I'll add in the CSS because I'm 5/5 that it needs margin.

Reply:
In some ways, a `&nbsp;` is elegant because it scales with the font, and since the icon is a font-icon, it will automatically provide margin that is appropriate for the font size even if browser is zoomed, etc.

Reply:
You could check with Asaad on what he'd prefer since he mostly owns the CSS, but we rarely use nbsp, so I'd assume he'd prefer a margin. I think we may just use px sizes generally though, so I don't know if scaling with font is something we care that much about.

Reply:
Ok, I'll check with @asaadmahmood

Reply:
@bradjcoughlin I would suggest that you just use `margin-right` class.
We can actually use margins with em's as well, and that would make the spacing relative to the font, but we're not doing that yet, and we may avoid that aswell, because we want to keep our spacing consistent rather than it being relative to the font.